src/components/board/board.test.js

```diff
@@ -1,8 +1,36 @@
+const board = require('../../services/boardService');
 
+const FILES = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'];
+const RANKS = [1, 2, 3, 4, 5, 6, 7, 8];
 
+// testing to make sure the array is populated with 64 object values
+test('Function will create an array of square objects', () => {
+  expect (board(FILES, RANKS)).toHaveLength(64);
+});
+
+// testing to make sure thhe objects have the required grid properties
+test('Function will have objects with chessboard grid property values ', () => {
```

Comment:
Not sure the difference between these two tests - the description is the same but the expectation is different as far as the id goes. I think it would be better to set the board variable to something static here and then call the makeBoard function directly in the test so you isolate the function your testing rather than testing it indirectly through a variable setting. I'll look through this test more and see if I can come up with an example to help out.

Reply:
my initial reason to make it that way was just checking to make sure the first coordinate is thee, and then checking that another random value thats supposed to be in there is, not just the first, I should have been more descriptive in my describe, but we can def look into your method at some point

Reply:
Ah - I see - that makes sense. You can do that in the first test. Check a few random squares. 

Then check some tings that could make this function fail. Like reversing the FILES and RANKS and make sure you're handling that situation. Think of other things that can be tested that you may need to care for.